core/src/script/CGXP/api/Map.js

```diff
@@ -359,7 +359,19 @@ cgxp.api.Map.prototype = {
         }
         this.map.setCenter(new OpenLayers.LonLat(center[0], center[1]), zoom);
     },
-
+    
+    /** api: method[recenterOnBoundingBox]
+     *  allow to recenter on a bounding box 
+     *  :arg minx: ``Number`` minx coordinate
+     *  :arg miny: ``Number`` miny coordinate
+     *  :arg maxx: ``Number`` maxx coordinate
+     *  :arg maxy: ``Number`` maxy coordinate
+     */
+    recenterOnBoundingBox: function(minx, miny, maxx, maxy) {
+        var OpenLayersBB = new OpenLayers.Bounds(minx, miny, maxx, maxy);
```

Comment:
Looks good, but the variable should start with a lower case, in this case `bbox` is a good name :-)